tests/integration/defs/s3_output.py

```
@@ -0,0 +1,245 @@
+                    section_name,
+                    f"{filesize} bytes uploaded to {fileurl}",
+                )
+            )
+        except Exception as e:
+            logger.warning(
+                f"Upload failed. test_name: {test_name}, filename: {filename}, error: {e}"
+            )
+            with open(filepath, "r", encoding="utf-8") as f:
+                limit = 65536
+                # Limit content to 64k (65536 bytes)
+                trail_content = f"... [truncated]"
```

Flake8 (7.3.0)

[error] f-string is missing placeholders

(F541)